Hudl.Mjolnir.Tests/Command/Attribute/NewCommandAttributeAndProxyTestsIgnoringTimeouts.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using Hudl.Mjolnir.Command.Attribute;
+using Hudl.Mjolnir.Tests.Helper;
+using Xunit;
+
+namespace Hudl.Mjolnir.Tests.Command.Attribute
+{
+    public sealed class NewCommandAttributeAndProxyTestsIgnoringTimeouts : TestFixtureIgnoreTimeouts
```

Comment:
The `TestFixtureIgnoreTimeouts` caught me here - I was reading through the tests and said to myself, "it looks like these aren't actually using the `ignoreTimeouts` property", and I stumbled upon how it was used later. What are your thoughts on bringing the config value closer to the tests here? It feels too disconnected to me.